documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1439,6 +1439,17 @@ See {link-prefix}:{link-avro-serialization}#avro-naming[Avro naming] for more de
 
 See {link-prefix}:{link-oracle-connector}#oracle-transaction-metadata[Transaction Metadata] for additional details.
 
+|[[oracle-property-log-mining-strategy]]<<oracle-property-log-mining-strategy, `+log.mining.strategy+`>>
+|`redo_log_catalog`
+|The mining strategy controls how Oracle LogMiner builds and uses a given data dictionary for resolving table and column ids to names. +
```

Comment:
When you say "Oracle LogMiner", do you mean "the LogMiner-based connector implementation"?